code/ATMOSPHERICS/components/unary_devices/vent_pump.dm

```diff
@@ -339,6 +340,39 @@
 		else
 			user << "<span class='notice'>You need more welding fuel to complete this task.</span>"
 			return 1
+	if(istype(W, /obj/item/weapon/screwdriver))
+		if(!welded)
+			if(open)
+				user << "\blue Now closing the vent."
+				if (do_after(user, 20, target = src))
+					open = 0
+					user.visible_message("[user] screwdrivers the vent shut.", "You screwdriver the vent shut.", "You hear a screwdriver.")
+			else
+				user << "\blue Now opening the vent."
+				if (do_after(user, 20, target = src))
+					open = 1
+					user.visible_message("[user] screwdrivers the vent shut.", "You screwdriver the vent shut.", "You hear a screwdriver.")
+		return
+	if(istype(W, /obj/item/weapon/paper/))
```

Comment:
Alright! Looks like I'll need to remember that the slash doesn't do anything, in the future - for some reason, I thought it did.